packages/terra-doc-template/src/IndexPageTemplate.jsx

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
```

Comment:
SOOOO, when I created this component in terra-dev-site way back, I chose the name `IndexPageTemplate` which does not accurately reflect what this component does. Lets rename this to `ExampleTemplate`